config/ROOT.scons

```diff
@@ -28,7 +28,7 @@ rootdict = Builder(generator = generate_action,
 rootenv = Environment(ENV={'PATH' : os.environ['PATH']})
 rootenv.ParseConfig("root-config --cflags --ldflags --libs")
 rootenv.Append(LIBS='-lSpectrum')
-rootenv.Append(LIBS=['-lFoam', '-lPyROOT'])
+rootenv.Append(LIBS=['-lFoam', '-lPyROOT','-lMinuit2'])
```

Comment:
This is a new dependency, requiring ROOT to be configured with `--enable-minuit2`... or at least that used to be the case. I think that's fine, but could you add a note in the installation guide?